Day1/P1.py

```diff
@@ -0,0 +1,300 @@
+#string capitalize method: change first character in a string to upper case, while other characters remain lower case
+from gettext import find
+
+
+a = "hello, how are you?"
+print(a.capitalize())
+
+def capitalizestr(txt):
+    if txt[0] == 'a':
+        new_txt = 'A' + txt[1:]
```

Comment:
This line is repeated all the time and the only change is the character to be used. Can you generalize this? For example, can you put the character into some kind of collection (list/dictionary/sets...) that will allow you to find the correct character to substitute rightaway?